components/LessonCard.tsx

```diff
@@ -26,7 +22,7 @@ const ReviewCount: React.FC<ReviewCountProps> = props => {
   }
 
   if (!data) {
-    return null
+    return <span> </span>
```

Comment:
What's the purpose of returning `<span></span>` vs `null`?

Reply:
Since the `ReviewCount` component represents a text value (rendered in a `span`), and its rendered in the parent component `ReviewButton` between two words (with no additional, surrounding spaces), when it returns `null` the two words have no space between them (leading to the condition cited in the original issue).

There are other ways to accomplish this. I might write this very differently if I was starting from scratch. An HOC that handles the query and passes relevant props down to `LessonCard` might be the way to go.

Reply:
I see. Maybe we can just add a comment to summarize why it's a span, just to make sure someone doesn't accidentally change it back to null lol

Reply:
But it appears that the query might need to be modified to return something other than null so we have a clearer representation of what's happening.

Reply:
I appreciate your feedback. We have some bigger changes to make at some point in the future, but in the mean-time returning an empty `span` isn't much of an improvement over `null`.

This component is called `ReviewCount`, so aside from the spinner (let's not go there for now 😀), this component should return a review count. I'll update it to return `0` in the odd/rare case where the query data is `undefined`. That should bring a sliver of consistency to the component.

Please take a look after the next commit is applied and let me know if you think a clarifying comment is still in order.